Doc/library/decimal.rst

```diff
@@ -1352,7 +1352,7 @@ In addition to the three supplied contexts, new contexts can be created with the
       Return ``x`` to the power of ``y``, reduced modulo ``modulo`` if given.
 
       With two arguments, compute ``x**y``.  If ``x`` is negative then ``y``
-      must be integral.  The result will be inexact unless ``y`` is integral and
+      must be integer.  The result will be inexact unless ``y`` is integer and
```

Comment:
Here and in lots of other places in this PR, the new text isn't grammatically correct. The word "integer" is a noun, but here you're trying to use it as an adjective. "must be *an* integer" would work, but the existing "integral" is fine, too - I don't see a good reason to change this.

I'd strongly recommend splitting this PR up into two separate PRs: one for the `Integral` -> `Integer` change, and one for the mass of grammatical changes. Those two sets of changes need quite different review processes.